docs: add security advisory 018 DOC-2943 (#10822)
spectrocloud/librarium: docs/docs-content/security-bulletins/security-advisories/security-advisories.md

:::

## Security Advisory 018 - Multiple Container Runtime Vulnerabilities

- **Release Date**: June 23, 2026

> - **Last Updated**: June 23, 2026

Reviewer: ```suggestion
- **Release Date**: June 24, 2026
- **Last Updated**: June 24, 2026
```

Since the release is delayed

Author: Added manually.

- **CVEs**:
  - [CVE-2026-50195](#cve-2026-50195---improper-validation-during-container-restoration)
  - [CVE-2026-53488](#cve-2026-53488---improper-handling-of-container-image-metadata)
  - [CVE-2026-53492](#cve-2026-53492---improper-validation-of-restoration-metadata)
  - [CVE-2026-53489](#cve-2026-53489---arbitrary-host-file-read-via-symlinked-log-paths)
  - [CVE-2026-47262](#cve-2026-47262---resource-exhaustion-leading-to-service-disruption)

### Overview

Multiple vulnerabilities have been identified in [containerd](https://github.com/containerd/containerd) that could allow
attackers to achieve remote code execution, privilege escalation, information disclosure, denial of service, and bypass

> of Kubernetes security controls.

Reviewer: ```suggestion
Kubernetes security controls.
```

Author: Added manually

#### Impact

- Arbitrary host command execution
- Potential node compromise
- Escalation from image author to node-level access

None of the annotations required for exploitation are enabled by default on Palette deployments or workload clusters.

### CVE-2026-53492 - Improper Validation of Restoration Metadata

- **Severity**: MEDIUM
- **CVSS Score**: 6.8

A vulnerability affecting container restoration workflows may allow security controls governing device and resource
access to be bypassed. Under certain circumstances, an attacker could gain access to resources that would otherwise be
restricted.

#### Impact

- Unauthorized device access
- Host mount injection
- Bypass of Kubernetes device enforcement policies

### CVE-2026-53489 - Arbitrary Host File Read via Symlinked Log Paths

- **Severity**: MEDIUM
- **CVSS Score**: 6.5

A flaw in restoration-related file handling may allow unauthorized access to information stored on the host system.
Successful exploitation could result in exposure of sensitive data.

#### Impact

- Arbitrary host file read
- Exposure of sensitive configuration or credential data

> ### CVE-2026-47262 - Resource Exhaustion Leading to Service Disruption

Author: ```suggestion
### CVE-2026-47262 - Resource Exhaustion Leading to Service Disruption
```

- **Severity**: MEDIUM
- **CVSS Score**: 6.8

A vulnerability in image processing may allow an attacker to cause excessive resource consumption on an affected node.
Successful exploitation could disrupt container management services and impact workload availability.

#### Impact

- Node-level denial of service
- Container startup and management failures
- Potential disruption of all workloads on the affected node

### Affected Deployments

#### Workload Clusters

- Managed Kubernetes clusters (EKS, AKS, GKE)
- PXK clusters managed through Palette
- RKE2 and K3s clusters managed through Palette
- Palette Enterprise and Palette VerteX deployments

#### SaaS Deployments

> - Self-hosted deployments

Reviewer: Are these supposed to be two separate bullets? If so, may be better to, instead of headings for each cluster/deployment, do a bulleted list like:

- SaaS deployments
- Self-hosted deployments
- Workload clusters that meet any of the following criteria:
    - Managed Kubernetes clusters (EKS, AKS, GKE)
    - PXK clusters managed through Palette
    - RKE2 and K3s clusters managed through Palette
    - Palette Enterprise and Palette VerteX deployments

Author: Yea it was silly to do the headings. I've made bullets.

### Mitigation and Remediation

#### SaaS Deployments

> - Multi-tenant and managed Dedicated SaaS clusters are being reviewed and patched as part of the standard update

Reviewer: ```suggestion
- Multi-tenant and dedicated SaaS clusters are being reviewed and patched as part of the standard update
```

Author: Change made manually.

  process. Continuous compliance scans detect unpatched nodes.

#### Self-Hosted Palette Enterprise and Palette VerteX Deployments

> - Deployments on cloud-managed Kubernetes, such as EKS, should update the clusters with the fixes provided by the cloud

Reviewer: ```suggestion
- Self-hosted installations deployed on managed Kubernetes clusters, such as EKS, should update the clusters with the fixes provided by the cloud
```

Reviewer: Wait, workload clusters deployed on managed K8s services via self-hosted? Otherwise the OS patching link doesn't make sense. (Though the link also calls out "This operation is not available for managed Kubernetes Services such as EKS and AKS. For EKS clusters, you can trigger OS updates from Palette. This would request AWS to update cluster node groups to the latest patch version." 😵)

Author: Yes I think Palette itself is deployed on EKS. I have removed this link.

  vendor. This requires patching the OS on cluster nodes. For guidance on patching cluster nodes, refer to

>   [OS Patching](../../clusters/cluster-management/os-patching.md).

Reviewer: ```suggestion
  vendor. This requires patching the OS on cluster nodes.
```

If this is for the management cluster and not workload clusters on self-hosted, the OS patching link doesn't describe how to do that.

Author: I will remove this link I think they didn't realise that it was for workload.

> - Deployments on customer infrastructure must upgrade to Kubernetes versions that include the patches.

Reviewer: So this is talking about the infra for the management cluster?
